maxminddb/reader.py

```diff
@@ -280,6 +248,43 @@ def _resolve_data_pointer(self, pointer: int) -> Record:
         (data, _) = self._decoder.decode(resolved)
         return data
 
+    def _load_buffer(
+        self, database: AnyStr | int | PathLike | IO, mode: int = MODE_AUTO
+    ) -> None:
+        self.filename: Any
```

Comment:
Rather than making this a public attribute, what do you think about returning it from this function?

Reply:
Fine for me, changed it